src/ogc/openSearch/OpenSearchRequest.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2015-2018 WorldWind Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @exports OpenSearchRequest
+ */
+
+define([
+        './OpenSearchConstants'
+    ],
+    function (OpenSearchConstants) {
+
+        /**
+         * Constructs a request object with the specified options.
+         * @alias OpenSearchRequest
+         * @constructor
+         * @classdesc Object representing an OpenSearch request.
+         * @param {Object} options
+         */
+        var OpenSearchRequest = function (options) {
```

Comment:
Please add comments on the public methods addHeader and getHeader. The properties of this class should also be documented as such.